contracts/evoting/types/transactions.go

```diff
@@ -385,3 +385,87 @@ func (removeAdmin RemoveAdmin) Serialize(ctx serde.Context) ([]byte, error) {
 
 	return data, nil
 }
+
+// AddVoter defines the transaction to Add an Voter
+//
+// - implements serde.Message
+type AddVoter struct {
+	// FormID is hex-encoded
+	FormID string
+	UserID string
+}
+
+// Serialize implements serde.Message
+func (addVoter AddVoter) Serialize(ctx serde.Context) ([]byte, error) {
```

Comment:
the four Serialize functions are almost the same, maybe you can call some function implementing the common code instead?